evennia/comms/channelhandler.py

```diff
@@ -43,6 +43,8 @@ class ChannelCommand(command.Command):
     Usage:
        {lower_channelkey}  <message>
        {lower_channelkey}/history [start]
+       {lower_channelkey} off - mutes the channel
```

Comment:
I think this should really somehow be shown in the return of the `channels` command, maybe as channelname (mute) or some such.